lib/6to5/transformation/transformers/playground-mallet-operator.js

```diff
@@ -0,0 +1,86 @@
+var t = require("../../types");
+
+var isMallet = function (node) {
+  var is = t.isAssignmentExpression(node) && node.operator === "||=";
+  if (is) {
+    var left = node.left;
+    if (!t.isMemberExpression(left) && !t.isIdentifier(left)) {
+      throw new Error("Expected type MemeberExpression or Identifier");
```

Comment:
You can use `throw file.errorWithNode(node, "error message here");` and it'll output a syntax error at that exact node position.